cmd/ddev/cmd/remove.go

```diff
@@ -20,35 +21,33 @@ leave database contents intact. Remove never touches your code or files director
 
 To remove database contents, you may use the --remove-data flag with remove.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		var siteName string
-
-		if len(args) > 1 {
-			util.Failed("Too many arguments provided. Please use 'ddev remove' or 'ddev remove [appname]'")
-		}
-
-		if len(args) == 1 {
-			siteName = args[0]
+		// Prevent users from destroying everything
+		if removeAll && removeData {
+			util.Failed("Illegal option combination: --all and --remove-data")
 		}
 
-		app, err := ddevapp.GetActiveApp(siteName)
+		apps, err := getRequestedApps(args, removeAll)
 		if err != nil {
-			util.Failed("Failed to remove: %v", err)
+			util.Failed("Unable to remove project(s): %v", err)
 		}
 
-		if app.SiteStatus() == ddevapp.SiteNotFound {
-			util.Failed("Project is not currently running. Try 'ddev start'.")
-		}
+		// Iterate through the list of apps built above, removing each one.
+		for _, app := range apps {
+			if app.SiteStatus() == ddevapp.SiteNotFound {
+				util.Warning("Project is not currently running. Try 'ddev start'.")
+			}
 
-		err = app.Down(removeData)
-		if err != nil {
-			util.Failed("Failed to remove %s: %s", app.GetName(), err)
-		}
+			if err := app.Down(removeData); err != nil {
+				util.Warning("Failed to remove %s: %s", app.GetName(), err)
```

Comment:
Looks to me like we're going to fall through after this and say that everything succeeded?

Reply:
Good point, these have been updated to not imply success.